tools/generator/src/Generator/AddBazelDependenciesTarget.swift

```diff
@@ -226,15 +224,11 @@ perl -pe 's/\$(\()?([a-zA-Z_]\w*)(?(1)\))/$ENV{$2}/g' \
     private static func createBuildScript(
         in pbxProj: PBXProj,
         name: String,
-        scriptPath: FilePath,
-        filePathResolver: FilePathResolver
+        script: String
     ) throws -> PBXShellScriptBuildPhase {
-        let scriptPath = try filePathResolver
-            .resolve(scriptPath, mode: .script)
-            .string
         let script = PBXShellScriptBuildPhase(
             name: "\(name) Run Script",
-            shellScript: #""\#(scriptPath)""#,
+            shellScript: script,
```

Comment:
Let's add a new line to this, since Xcode prefers a trailing newline, and people most likely won't include one in their normal one-line usage.